data/hzpeng57.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "HuangZhenpeng",
+  "bio": "Front End Development Engineer",
+  "tags": ["Typescript", "Vue.js"],
+  "links": [
+    {
+      "name": "GitHub",
+      "url": "https://github.com/hzpeng57",
+      "icon": "FaGithub"
+    },
+    {
+      "name": "LinkedIn",
+      "url": "https://www.linkedin.com/in/hzpeng/"
```

Comment:
Hi, you have a comma (**,**) missing on **line 13,** rendering the json file invalid.